src/SSLLocks.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "murmur_pch.h"
+
+#include "SSLLocks.h"
+
+static QMutex **locks = NULL;
```

Comment:
Looks like we leak these. Why not a std::vector<QMutex> instead? (boost::ptr_vector is an alternative that would retain memory layout/semantics. Don't think we need to care about that though. Do we?)

As mentioned on IRC I'm a big fan of leak-free execution so you know that if you see a leak it actually is a problem. Then again I have no clue what the Qt policy is on that to begin with. In non Qt C++ & Boost applications I had very good results with leak sanitizer based on such a zero leaks policy.